version/1/4/it/code_of_conduct.md

```diff
@@ -0,0 +1,47 @@
+# Patto di Condotta dei Collaboratori
+
+## Il Nostro Impegno
+
+Nell'interesse di incoraggiare un ambiente sociale aperto e accogliente, noi in quanto collaboratori e responsabili ci impegniamo a rendere la partecipazione al nostro progetto e alla nostra comunità un'esperienza priva di molestie per tutti, indipendentemente da età, dimensioni del corpo, disabilità, etnia, identità ed espressione di genere, livello di esperienza, nazionalità, aspetto fisico, razza, religione, o identità e orientamento sessuale.
+
+## I Nostri Standard
+
+Esempi di comportamento che contribuiscono a creare un ambiente sociale positivo includono:
+
+* Usare un linguaggio accogliente e inclusivo
+* Essere rispettosi di punti di vista ed esperienze differenti
+* Accettare in modo composto critiche costruttive
+* Concentrarsi su cosa è meglio per la comunità
+* Mostrare empatia verso gli altri membri della comunità
+
+Esempi di comportamento inaccettabile da parte dei partecipanti includono:
+
+* L'uso di linguaggio o immagini sessuali e attenzioni o advances sgraditi
+* Comportamento da troll o commenti offensivi/dispregiativi, e attacchi personali o politici
+* Molestie pubbliche o private
+* Pubblicazione di informazioni private di un individuo, quali l'indirizzo reale e/o elettronico, senza l'esplicito consenso
+* Altri comportamenti che possono ragionevolmente considerati inappropriati in ambito professionale
+
+## Le Nostre Responsabilità
+
+I responsabili del progetto sono tenuti a chiarire gli standard di comportamento accettabile e a prendere azioni correttive appropriate e giuste in risposta a qualsiasi caso di comportamento inaccettabile.
+
+I responsabili del progetto hanno il diritto e la responsabilità di rimuovere, modificare, o cancellare commenti, commit, codice, modifiche del wiki, ticket, ed altri contributi che non sono in linea con questo Patto di Condotta, o di bandire temporaneamente o permanentemente qualsiasi collaboratore per altri comportamenti ritenuti inappropriati, intimidatori, offensivi o dannosi.
```

Comment:
This is a little remaining of 1.3 translation...
It's not really an error, because, if I'm not wrong, Italian is quite flexible in these cases.

We should also talk about this 👍 
@amencarini